ds3-autogen-java/src/main/resources/tmpls/response/get_objects_details_response_template.ftl

```diff
@@ -0,0 +1,58 @@
+<#include "../copyright.ftl"/>
+
+package ${packageName};
+
+<#include "common/response_imports.ftl"/>
+import java.util.List;
+
+public class ${name} extends AbstractResponse {
+
+${javaHelper.createAllResponseResultClassVars(responseCodes)}
+    private Integer pagingTruncated;
+    private Integer pagingTotalResultCount;
+
+<#include "common/response_constructor.ftl"/>
+
+    @Override
+    protected void processResponse() throws IOException {
+        try {
+            this.checkStatusCode(${helper.getResponseCodes(responseCodes)});
+
+            switch (this.getStatusCode()) {
+            case 200:
+                try (final InputStream content = getResponse().getResponseStream()) {
+                    this.s3ObjectListResult = XmlOutput.fromXml(content, S3ObjectList.class);
+                    this.pagingTruncated = parseIntHeader("page-truncated");
+                    this.pagingTotalResultCount = parseIntHeader("total-result-count");
+                }
+                break;
+            default:
+                assert false : "checkStatusCode should have made it impossible to reach this line.";
+            }
+        } finally {
+            this.getResponse().close();
+        }
+    }
+
+    private Integer parseIntHeader(final String key) {
```

Comment:
Is this the only request that has pagination support like this?  If it is not this should be pulled out into a different template.